percy/providers/genericProvider.js

```diff
@@ -26,6 +26,7 @@ class GenericProvider {
   constructor(driver) {
     this.driver = driver;
     this.metadata = null;
+    this.debugUrl = null;
```

Comment:
In order to keep same structure of base and child class keep this class same.
i.e. define this variable but update existing `getDebugUrl` to return `this.debugUrl`

Child will remain same. Basically it keeps the structure open for others to override `getDebugUrl` if required. [ they can also update `this.debugUrl` if required as you are doing in setDebugUrl in child class